.github/workflows/ci.yml

```diff
@@ -104,7 +104,7 @@ jobs:
           uses: actions/checkout@v3
           with:
             repository: maticnetwork/matic-cli
-            ref: v0.3.0-dev
+            ref: arpit/pos-655-2
```

Comment:
do we need it like that branch?

Reply:
yes for now
we will be making changes again in upcoming releases